cpp/extensions/cloud_optics/mo_cloud_optics.h

```diff
@@ -256,7 +256,9 @@ class CloudOptics : public OpticalProps {
     using yakl::COLON;
     using yakl::intrinsics::size;
     using yakl::intrinsics::allocated;
-    using yakl::intrinsics::any;
+    #ifdef RRTMGP_EXPENSIVE_CHECKS
```

Comment:
Why are the using statements protected in ifdefs now? Is there any harm in including them?

Reply:
Yes, with the latest version of YAKL, there is no need to use the ifdef protection, I have restored to the previous version